### ISSUE TITLE
GKE Multitenant Blueprint - logging configuration

### ISSUE THREAD

**Issue by olliefr** [GoogleCloudPlatform/cloud-foundation-fabric#1639]
Hey,

While working on #1638 I had noticed that the [`README` for GKE Multitenant Blueprint](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/2e0474dc71c0059d2750f942b6307678714450a5/blueprints/gke/multitenant-fleet/README.md) states:

https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/2e0474dc71c0059d2750f942b6307678714450a5/blueprints/gke/multitenant-fleet/README.md?plain=1#L16

My interpretation of this is such that both `SYSTEM_COMPONENTS` and `WORKLOADS` logs are being ingested as per [`LoggingComponentConfig`](https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1/projects.locations.clusters#loggingcomponentconfig). 

The examples in the blueprint README do not include any logging configuration so it falls back to the default value, which is defined to be system logs only:

https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/2e0474dc71c0059d2750f942b6307678714450a5/blueprints/gke/multitenant-fleet/variables.tf#L36

I think this is a divergence from what's stated in the README and either the README or the default value must be corrected.

I'm interested to know what, in your opinion, the answer should be so that I can update #1638 🙃 

## Default logging in Cloud Console vs [`gke-cluster-standard`](../../../modules/gke-cluster-standard)

I think this might be a consequence of a bigger point although I am not sure if it warrants a separate issue - in Cloud Console, pre-selected values for logging configuration are system logs and workloads, indeed.

In [`gke-cluster-standard`](../../../modules/gke-cluster-standard) as it stands today, the default value is system logs only.

I'm not sure if this divergence is intentional.

**Comment by olliefr** [comment]
While working on updating the example for using the new `logging_config` interface in #1638, I'd updated the example to also include `WORKLOADS` in the logs source. 

Thus, once #1638 is merged there is going to be no discrepancy between the README and the code.

Commit: fe13131783f053487f9414385488840370509d99
